packages/jbrowse-web/src/plugins/BigWigAdapter/util.js

```diff
@@ -46,7 +46,7 @@ export function rectifyStats(s) {
 export function calcRealStats(region, features) {
   const { start, end } = region
   const scores = []
-  const feats = features.sort((a, b) => a.start < b.start)
+  const feats = features.sort((a, b) => a.start - b.start)
```

Comment:
@cmdcolin This test wasn't passing, but this change seems to fix it. Just wanted to make sure this is still doing what you intended.

Reply:
that is definitely the proper way to sort numbers, looks good!